examples/amd-azure/README.md

```diff
@@ -0,0 +1,100 @@
+<!--
+Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Benchmarking
+
+The command-line (CLI) tool `multi-process-benchmark.py` is used to measure the performance using multiple separate processes in parallel. This is ideal for measuring performance on systems that have multiple sockets or multiple L3 caches, like AMD's Milan.
+
+## Quickstart
+
+After you have installed `deepsparse` in your Python environment, the benchmark tool is available on your CLI. It is simple to run expressive model benchmarks on the DeepSparse Engine with minimal parameters.
+
+To run parallel inferences of a sparse FP32 MobileNetV1 on each CCX of a Microsoft Azure HB120rs\_v3 at batch size 16 for 30 seconds for throughput:
+
```

Comment:
Could we also mention that the following command should be run from inside `examples/amd-azure` directory?